src/ui/marker.js

```diff
@@ -180,6 +184,7 @@ export default class Marker extends Evented {
             page1.appendChild(circleContainer);
 
             svg.appendChild(page1);
+            svg.setAttributeNS(null, 'transform', `scale(${this._scale})`);
```

Comment:
`setAttributeNS` has unknown support in IE11, it would be good to verify if it works at the root SVG element or switch to using `setAttribute` if viable.

Reply:
Makes sense @asheemmamoowala -- I went with `setAttributeNS` rather than `setAttribute` for consistency with the usage [here](https://github.com/mapbox/mapbox-gl-js/blob/b238b15f64e00540c29d0ec3d2065861aa90d1f9/src/ui/marker.js#L95-L100). Just to be sure, would you recommend testing with IE11 or switching to `setAttribute` before moving forward?